### PR TITLE
[ALLUXIO-2461] Add -d option in ls command to list dir as plain file

### PR DESCRIPTION
**Pull request by ifcharming**
https://alluxio.atlassian.net/browse/ALLUXIO-2461

This makes it possible to `ls` the metadata for root dir `"/"`:
`./bin/alluxio fs ls -d / `

### CODE REVIEW

**Comment by ifcharming** [comment]
@apc999 @aaudiber

File: tests/src/test/java/alluxio/shell/command/LsCommandTest.java [hunk 1]
```diff
@@ -102,6 +102,23 @@ public void lsNoAcl() throws IOException, AlluxioException {
   }
 
   /**
+   * Tests ls -d command when security is not enabled.
+   */
+  @Test
+  @LocalAlluxioClusterResource.Config(
+      confParams = {PropertyKey.Name.SECURITY_AUTHORIZATION_PERMISSION_ENABLED, "false",
+          PropertyKey.Name.SECURITY_AUTHENTICATION_TYPE, "NOSASL"})
+  public void lsDirectoryAsPlainFileNoAcl() throws IOException, AlluxioException {
```
Comment (apc999) [inline tests/src/test/java/alluxio/shell/command/LsCommandTest.java]: can you add a test for root?

**Comment by apc999** [comment]
minor comments left.

Thanks for the improvement.

File: tests/src/test/java/alluxio/shell/command/LsCommandTest.java (hunk 1 above)
Comment (ifcharming) [inline tests/src/test/java/alluxio/shell/command/LsCommandTest.java]: added.

File: shell/src/main/java/alluxio/shell/command/LsCommand.java [hunk 2]
```diff
@@ -148,18 +162,19 @@ public int compare(URIStatus status1, URIStatus status2) {
...
   @Override
   public void runCommand(AlluxioURI path, CommandLine cl) throws AlluxioException, IOException {
-    ls(path, cl.hasOption("R"), cl.hasOption("f"));
+    ls(path, cl.hasOption("R"), cl.hasOption("f"), cl.hasOption("d"));
   }
 
   @Override
   public String getUsage() {
-    return "ls [-R] [-f] <path>";
+    return "ls [-R] [-f] [-d] <path>";
```
Comment (ifcharming) [inline shell/src/main/java/alluxio/shell/command/LsCommand.java]: done.

**Comment by ifcharming** [comment]
Thanks @apc999 for the review. Can you PTAL?

**Pull request merged**